src/popup/components/Header.vue

```diff
@@ -37,10 +37,12 @@
       v-if="isLoggedIn"
       class="right"
     >
-      <NotifyBell />
+      <NotifyBell
+        v-if="showNotifications"
+      />
```

Comment:
The style of the project is such that one attribute should not be on the next line.
Just like in that example. 
https://eslint.vuejs.org/rules/max-attributes-per-line.html

Reply:
Yeah, I know. But there are 2 elements after this one with the v-if directives in separate lines so I wanted to maintain same flow here. But sure, I'll change this.